packages/angular/cli/src/command-builder/schematics-command-module.ts

```diff
@@ -204,14 +204,15 @@ export abstract class SchematicsCommandModule
 
                   return definition.validator(Object.values(values).map(({ value }) => value));
                 },
-                default: definition.default,
+                default: definition.multiselect ? undefined : definition.default,
                 choices: definition.items?.map((item) =>
                   typeof item == 'string'
                     ? {
                         name: item,
                         value: item,
                       }
                     : {
+                        ...item,
```

Comment:
Include options that are in line with the new API: https://github.com/SBoudrias/Inquirer.js/tree/main/packages/checkbox#choice-object